libdispatch/dattput.c

```diff
@@ -83,9 +83,9 @@ be assigned or ::NC_GLOBAL for a global attribute.
 \param name Attribute \ref object_name. \ref attribute_conventions may
 apply.
 
-\param len Number of values provided for the attribute.
+\param len Length of `value` not including trailing null.
 
-\param value Pointer to one or more values.
+\param value Pointer to text value.
```

Comment:
Well in C there is no "text value". How about:

Pointer to an array of char with one or more elements; the array is the text value of the attribute.

Reply:
Well, the argument is `value` and it is `text` (hence the `nc_put_att_text`).  But I'm fine with your suggestion even though it has `text value` also...